spec/rubocop/cop/style/ternary_operator_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Style::TernaryOperator do
+  subject(:cop) { described_class.new }
+
+  it 'registers offense when the if branch and the else branch are ' \
+     'on a separate line from the condition' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ?
+        b : c
+    RUBY
+    expect(cop.offenses.size).to eq(1)
+  end
+
+  it 'registers an offense when the false branch is on a separate line' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ? b :
+          c
+    RUBY
+
+    expect(cop.offenses.size).to eq(1)
+  end
+
+  it 'registers an offense when everything is on a separate line' do
+    inspect_source(<<-RUBY.strip_indent)
+      a = cond ?
+          b :
+          c
```

Comment:
I think this cop should not add offence to multiline ternary operator. Because coverage is calculated for a ternary operator if the operator is on multiline.

![170807001543](https://user-images.githubusercontent.com/4361134/29004626-9b6440d8-7b05-11e7-9d4a-ac1bafa35001.png)